cmd/istio-cni/main_test.go

```diff
@@ -207,6 +207,7 @@ func TestCmdAddTwoContainers(t *testing.T) {
 
 	setupRedirect = mockNsenterRedirect
 	testAnnotations[injectAnnotationKey] = "true"
+	testAnnotations[sidecarStatusKey] = "true"
```

Comment:
Should `testAnnotations[sidecarStatusKey]` be set to `true` by default and only set to `false` in the `TestCmdAddTwoContainersWithoutSideCar` test?

Reply:
Hi @venilnoronha I'm not sure that this will work, The value of the annotation will be set to false, but the annotation is still set so it would still try and redirect as the annotation is present, even without the sidecar.

Reply:
My concern here is that all tests might be affected by the `excludePod` logic based on the status annotation. Can we still set the annotation by default and remove it just for a specific test?